lib/view/sfViewCacheManager.class.php

```diff
@@ -29,7 +29,8 @@ class sfViewCacheManager
     protected $routing;
     protected $request;
     protected $loaded = array();
-
+    protected $options = array();
```

Comment:
Dynamic property are public.

So It is a BC break to reduce its accessibility scope.

I recommend making it `public`.

Reply:
Thanks for your feedback I have just made the correction.

Reply:
@alquerci Ups, just to let you know: We have done many of those changes for 8.1 and 8.2 compat in the past.